src/exchange/tradingPair/TradingPairView.scss

```diff
@@ -60,36 +64,37 @@
 }
 
 .dropdownBtnActive {
-    box-shadow: inset 1px 1px 7px 2px rgba(0, 0, 0, 0.3);
+    background-color: #35353E;
+    border-bottom-left-radius: unset;
 }
 
-.dropdownList {
+.dropdownListWrapper {
     position: absolute;
     z-index: 10;
-    width: 100%;
     top: 48px;
+    overflow: hidden;
+    margin: 0 -2px -2px -3px;
+}
+
+.dropdownList {
+    display: table;
     list-style: none;
-    padding: 0;
-    margin: 0;
+    padding: 10px 0 0 0;
+    margin: 0 2px 2px 2px;
 
-    background-color: $grey-darkest;
+    background-color: #35353E;
     border: 1px solid $grey-darken-darkest;
     border-top: none;
-    box-shadow: 0 0 4px 1px rgba(black, 0.20);
+    box-shadow: 0 0 4px 1px rgba(0, 0, 0, 0.2);
```

Comment:
Why did you changed that exactly?

Reply:
Interesting. Perhaps copy-paste from the debugger